container/blat/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye-slim
+
+LABEL org.opencontainers.image.source="https://github.com/stjude/proteinpaint" \
+    org.opencontainers.image.version="latest"
+
+WORKDIR /home/root/blat
+ARG COMMAND=""
+ENV COMMAND=$COMMAND
+ENV PORT=1234
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    wget \
+    build-essential \
+    libcurl4-openssl-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/blat/gfServer \
+    && chmod +x gfServer
+
+RUN mkdir -p ./genomes
+
+EXPOSE $PORT
+
+CMD ["sh", "-c", "$COMMAND"]
```

Comment:
the variable "$COMMAND" here seems too permissive, may want to restrict this a bit. For example, 
```dockerfile
CMD ["sh", "-c", "./gfServer start $HOST $PORT -stepSize=$STEPSIZE /home/root/blat/genomes/$HGNUM.2bit"]
```

Reply:
I did like this:

`CMD ["sh", "-c", "./gfServer start $HOST $PORT -stepSize=$STEPSIZE /home/root/blat/genomes/$HGNUM.2bit"]`

Reply:
@aacic I think `$HOST` will always be `127.0.0.1` within the container? If yes, should hard code this instead of being an argument. But I approved the PR either way.

Reply:
Ok. It's hardcoded and merged.